assets/js/onboard-tasks.js

```diff
@@ -0,0 +1,146 @@
+/* global prplDocumentReady */
+/*
+ * Onboard tasks
+ *
+ * A script to process the onboarding task checklist.
+ *
+ * Dependencies: progress-planner-document-ready
+ */
+
+/**
+ * Process the onboarding task checklist.
+ */
+// eslint-disable-next-line no-unused-vars
+async function progressPlannerOnboardTasks() {
```

Comment:
We should probably rename this to `prplOnboardTasks` for consistency with other files